dataService.js

```diff
@@ -0,0 +1,53 @@
+// Getting data from json files
+const categoryData = require('./src/data/categories.json');
+const productData = require('./src/data/products.json');
+
+// Maps
+let productMap = {};
+let categoryMap = {};
+let combinedProductMap = [];
+
+function getProducts() {
+  productData.products.forEach(product => {
+    productMap[product.id] = product;
```

Comment:
Create a method variable, and assign that to `productMap` at last

Reply:
Created a function for that which returns the required map and then assigned it to `productMap`